arangod/RestHandler/RestStatusHandler.cpp

```diff
@@ -163,3 +178,66 @@ RestStatus RestStatusHandler::execute() {
   generateResult(rest::ResponseCode::OK, result.slice());
   return RestStatus::DONE;
 }
+
+RestStatus RestStatusHandler::executeOverview() {
+  VPackBuilder result;
+
+  result.add(VPackValue(VPackValueType::Object));
+  result.add("version", VPackValue(ARANGODB_VERSION));
+  result.add("platform", VPackValue(TRI_PLATFORM));
+
+#ifdef USE_ENTERPRISE
```

Comment:
Maybe we could create a function which adds similarities to the builder which occur in `executeOverview` and `executeStandard` instead of having duplicate code here.

Reply:
it is too different